sdk/storage/Azure.Storage.Common/tests/StreamPartitionerTests.cs

```diff
@@ -133,7 +132,7 @@ public async Task GetNextPartitionAsync_1GB()
                         }
                     }
 
-                    Assert.IsTrue(GC.GetTotalMemory(true) - memoryStart < 8 * Constants.DefaultBufferSize); // TODO Assuming at most 8 buffers allocated
+                    Assert.IsTrue(GC.GetTotalMemory(true) - memoryStart < 8 * Storage.Constants.DefaultBufferSize); // TODO Assuming at most 8 buffers allocated
```

Comment:
Why was this change necessary, out of curiosity?

Reply:
There is a different Constants file in Storage.Tests